app/build.gradle

```diff
@@ -119,20 +126,19 @@ dependencies {
         "androidx.legacy:legacy-support-v4:$android_legacy_support_version",
         "androidx.appcompat:appcompat:$android_support_version",
         "androidx.recyclerview:recyclerview:$android_support_version",
-        "com.google.android.material:material:$android_material",
+        "com.google.android.material:material:$android_material_version",
         "androidx.constraintlayout:constraintlayout:$constraint_layout_version",
 
-        "io.reactivex.rxjava2:rxjava:$rxjava_version",
-        "io.reactivex.rxjava2:rxandroid:$rxandroid_version",
-        "com.jakewharton.rxbinding2:rxbinding:$rxbinding_version",
+        "com.jakewharton.rxbinding3:rxbinding:$rxbinding_version",
+        "com.jakewharton.rxbinding3:rxbinding-core:$rxbinding_version",
```

Comment:
Thinking if this should be inside `common-rx` module or not

Reply:
@Tuubz I think we can keep it in `app` module since this only use in `app` and it is not mandatory 🙏